tests/tests/unit-tests/routers/OrganizationRouterTest.js

```diff
@@ -0,0 +1,93 @@
+var blueprint = require ('@onehilltech/blueprint')
+  , request   = require ('supertest')
+  , expect    = require ('chai').expect
+  ;
+
+var appPath         = require ('../../../fixtures/appPath');
+var organizations   = require ('../../../fixtures/organizations');
+
+describe ('OrganizationRouter', function () {
+  before (function (done) {
+    blueprint.testing.createApplicationAndStart (appPath, done)
+  });
+
+  after (function (done) {
+    blueprint.app.models.Organization.remove ({}, done);
+  });
+
+  describe ('/organizations', function () {
+
+    var organization = organizations[0];
+    var organizationId;
+
+    describe ('POST', function (done) {
+      it ('should create an organization in the database', function (done) {
+        request (blueprint.app.server.app)
+          .post ('/organizations') // route
```

Comment:
The route you are hitting is actually /v1/admin/organizations and will require an admins access_token to use.  Please look at the UserRouterTest and LoginRouterTest for how to set that up in a before statement.